src/main/features/directions-questionnaire/forms/models/selfWitness.ts

```diff
@@ -23,7 +23,7 @@ export class SelfWitness implements CompletableTask {
 
   deserialize (input?: any): SelfWitness {
     if (input && input.option) {
-      this.option = YesNoOption.fromObject(input.option)
+      this.option = YesNoOption.fromObject(input.option) || YesNoOption.fromObject(input.option.option)
```

Comment:
this doesn't seem right. looks like something needs renaming or returned differently